readme.md

```diff
@@ -105,6 +111,21 @@ String or array of strings to match.
 
 Type: `object`
 
+##### allPatterns
+
+Type: `boolean`\
+Default: `false`
+
+Requires all negated patterns to not match and any normal patterns to match at least once. Otherwise, it will be a no-match condition.
+
+This option may slow down `matcher.isMatch` with long inputs.
+
+**Example:** find text strings containing both "Tesla" and "supremacy" in arbitrary order, but not "hoax".
```

Comment:
Please find a more non-contentious example. Maybe animals or some objects.

Reply:
I hope "tiger" is more neutral than "tesla" :)